mungegithub/mungers/mungers.go

```diff
@@ -69,15 +69,22 @@ func RequestedFeatures() []string {
 	return out.List()
 }
 
-// InitializeMungers will call munger.Initialize() for all mungers requested
```

Comment:
Previously, the mungers would first be validated & initialized, before the necessary features were initialized. The order was:

- mungers register & initialize
- features initialize

However, submit-queue.go would use the GCS feature during its initialization, so, it was necessary to initialize the features before the mungers. Hence, I broke it up such that the order becomes:

- mungers are registered
- list of features is obtained
- features initialize
- mungers initialize